release_images.yml

```diff
@@ -2,10 +2,10 @@
 release_images:
   1:
     framework: "tensorflow"
-    version: "2.10.1"
+    version: "2.9.2"
```

Comment:
Could you please remove this change? Perhaps you need to rebase your commits onto @ShiboXing's r1.12.1 branch.

Reply:
This is because of my sync to master head: https://github.com/aws/deep-learning-containers/compare/master...sheilaliuxl:deep-learning-containers:smmp.

Can @ShiboXing sync your branch to head (as it'll need to sync eventually)? Thanks.

Reply:
@saimidu https://github.com/ShiboXing/deep-learning-containers/pull/3#discussion_r1046547517

Based on this, do you still need anything from us?

Reply:
This seems like the commit from master that's missing, that Sheila has added here 
https://github.com/ShiboXing/deep-learning-containers/compare/r1.12.1...aws:deep-learning-containers:master